### PR TITLE
Add missing engineers to the list

### PR DESCRIPTION
**Pull request by MatrixFrog**

### CODE REVIEW

**Comment by Filip-F** [comment]
Can you add me as well? `filip-f`

**Comment by Filip-F** [comment]
Also @ninachaubal is missing.

File: config/HustleInc.yml [hunk 1]
```diff
@@ -17,6 +17,8 @@ eng:
     - higgins
     - Lizard381
     - matthewferry
+    - MatrixFrog
+    - ibash
```
Comment (noahsilas) [inline config/HustleInc.yml:21]: This looks like it hasn't been updated in a while...
Nina is `ninachaubal`
Richard is `richardharrington`
Tim is `caplis`
Filip is `Filip-F`
King is `kingdavidmartins`
I don't think I've seen PRs from him, but we could add Bennett: `hustlerbb`

**Review by noahsilas — APPROVED** [review]

File: config/HustleInc.yml (hunk 1 above)
Comment (noahsilas) [reply config/HustleInc.yml:21]: Oh, also `robertbourbonnais`

Comment (Filip-F) [reply config/HustleInc.yml:21]: Is there no way to automatically pull this from the org so we don't have to do this manually every time?

Comment (MatrixFrog) [reply config/HustleInc.yml:21]: There probably is but I don't want to spend too much time on this. I will very easily fall into this trap if I'm not careful: https://xkcd.com/1629/

**Comment by MatrixFrog** [comment]
I have no idea how the permissions work but it won't let me merge this.

**Comment by noahsilas** [comment]
Ping @lklots : Looks like you have powers over this repo, can you merge this one?

File: config/HustleInc.yml (hunk 1 above)
Comment (ClaireNeveu) [reply config/HustleInc.yml:21]: I am also not on this list. We should probably add this to the onboarding instructions.

**Pull request merged**